src/OpenSage.Viewer/UI/Views/GameView.cs

```diff
@@ -56,6 +56,10 @@ public override void Draw(ref bool isGameViewFocused)
             {
                 isGameViewFocused = true;
             }
+            ImGui.SetCursorScreenPos(windowPos + new Vector2(1.0f));
+            ImGui.TextColored(Vector4.UnitW, $"{ImGui.GetIO().Framerate:N2} FPS");
```

Comment:
I didn’t know you can do formatting like this in an interpolated string. Very cool.

Reply:
It's neat, though I should have really cached the string instead of doing that twice for the dropshadow.